File: Part8/Python/FunctionConcepts/FunctionConcepts_13.py
# Python does not access arguments through main.
import sys

def main():
    print ( f"Number of arguments: {len(sys.argv)}" )
    print ( "Arguments:" )
    for i, arg in enumerate ( sys.argv ):
        print ( f"\tArgument {i}: {arg}" )

if __name__ == "__main__":
    main()

File: Part8/Go/FunctionConcepts/FunctionConcepts_13.go
// Go does not pass the function arguments to main, access via os.Args
package main

import (
    "fmt"
    "os"
)

func main() {
    fmt.Printf ( "Number of arguments: %d\n", len ( os.Args ) )
    fmt.Println ( "Arguments:" )
    for i, arg := range os.Args {
        fmt.Printf ( "\tArgument %d: %s\n", i, arg )
    }
}
